packages/react-notifications/package.json

```diff
@@ -54,10 +54,10 @@
   },
   "devDependencies": {
     "@commercetools-frontend/ui-kit": "10.1.2",
-    "@testing-library/react": "8.0.7",
+    "@testing-library/react": "9.1.0",
```

Comment:
<img width="861" alt="CleanShot 2019-08-12 at 09 37 45@2x" src="https://user-images.githubusercontent.com/1877073/62851129-6f28fa80-bce5-11e9-8cae-6cfcc7976100.png">

This is the main change. If this doens't break out tests we did a good job. Clean up is now automagic.